config.schema.json

```diff
@@ -45,21 +45,11 @@
         "default": "clean",
         "required": true
       },
-      "rooms" {
```

Comment:
I don't think removing them are OK. You can see there are some users setting up the plugin via the UI.

Reply:
ok, can you please help me with the schema then?

Reply:
isn't it just adding the missing `:` and removing the `default` entry? It's an optional parameter, if not defined, that's fine

Reply:
no ... please try yourself with this in it:
`"rooms": {
        "title": "Room cleaning",
        "type": "array",
        "required": false,
        "items": {
          "name": {
            "name": "Room Name",
            "type": "string"
          },
          "id": {
            "name": "Room ID",
            "type": "integer"
          }
        }
      }`

and you will see the next error

Reply:
Yeah... missing the`properties` attribute :)
```
    "rooms": {
        "title": "Room cleaning",
        "type": "array",
        "required": false,
        "items": {
          "type": "object",
          "properties": {
            "name": {
              "name": "Room Name",
              "type": "string"
            },
            "id": {
              "name": "Room ID",
              "type": "integer"
            }
          }
        }
      }
```

Reply:
oh great ... will update, gimme a minute

Reply:
Working! Great. Its in now :-)